config/client_config/dc/system/config/templates/features/aca_individual_market/eligibilities.yml

```diff
@@ -51,6 +51,7 @@ registry:
               - :citizenship
               - :american_indian_status
               - :social_security_number
+              - :dc_residency
```

Comment:
we need to find a different property name here--no keys or property names should reference a state specifically

can we use `state_residency` instead?

Reply:
perhaps this shouldn't be something that we use the resource registry for?

another option would be to use an environment variable, e.g.

`STATE_RESIDENCY_VERIFICATION=dc_residency` or `STATE_RESIDENCY_VERIFICATION=me_residency`

and then wherever you need to use that value, reference it with `ENV['STATE_RESIDENCY_VERIFICATION']`